src/Scanner.js

```diff
@@ -106,31 +108,19 @@ class DefaultScanner {
             // otherwise, use the delay time to defer the initial scan
             setTimeout(() => this.scanAndObserve(), delay)
         }
-        return () => {
-            const activeInput = this.device.activeForm?.activeInput
-
-            // remove Dax, listeners, timers, and observers
-            clearTimeout(this.debounceTimer)
-            this.mutObs.disconnect()
-
-            this.forms.forEach(form => {
-                form.resetAllInputs()
-                form.removeAllDecorations()
-            })
-            this.forms.clear()
-
-            // Bring the user back to the input they were interacting with
-            activeInput?.focus()
+        return (reason, ...rest) => {
+            this.stopScanner(reason, ...rest)
         }
     }
 
     /**
      * Scan the page and begin observing changes
      */
     scanAndObserve () {
-        window.performance?.mark?.('scanner:init:start')
+        window.performance?.mark?.('initial_scanner:init:start')
         this.findEligibleInputs(document)
-        window.performance?.mark?.('scanner:init:end')
+        window.performance?.mark?.('initial_scanner:init:end')
+        logPerformance('initial_scanner')
```

Comment:
I've created this `logPerformance` little utility to log performance metrics of every scan, not just the initial one. Works the exact same way.